libs/payments/stripe/src/lib/stripe.client.spec.ts

```diff
@@ -34,6 +34,25 @@ describe('StripeClient', () => {
     });
   });
 
+  describe('updateCustomer', () => {
+    it('updates an existing customer from Stripe', async () => {
+      const mockCustomer = CustomerFactory();
+      const mockUpdatedCustomer = CustomerFactory({
+        description: 'All your base are belong to us.',
```

Comment:
[non-blocking] You don't need to override a value here

Reply:
Q: don't I need to show an actual update though? To verify it actually worked?

Reply:
So the factory already generates a random value. You can use that for both the update call and the assertion.

```
      const mockCustomer = CustomerFactory();
      const mockUpdatedCustomer = CustomerFactory();

      mockClient.stripe.customers.update = jest
        .fn()
        .mockResolvedValueOnce(mockUpdatedCustomer);

      const result = await mockClient.updateCustomer(mockCustomer.id, {
        description: mockUpdatedCustomer.description,
      });

      expect(result.description).toEqual(mockUpdatedCustomer.description);
```

One of the reasons we want to write like this is to avoid hardcoded values, even in tests.